apps/api/src/app/integrations/usecases/create-novu-integrations/create-novu-integrations.usecase.ts

```diff
@@ -0,0 +1,75 @@
+import { Injectable } from '@nestjs/common';
+import { IntegrationEntity, IntegrationRepository } from '@novu/dal';
+
+import { CreateNovuIntegrationsCommand } from './create-novu-integrations.command';
+import { CreateIntegration } from '../create-integration/create-integration.usecase';
+import { CreateIntegrationCommand } from '../create-integration/create-integration.command';
+import { GetNovuIntegration } from '@novu/application-generic';
+
+@Injectable()
+export class CreateNovuIntegrations {
+  constructor(private createIntegration: CreateIntegration, private integrationRepository: IntegrationRepository) {}
+
+  async execute(command: CreateNovuIntegrationsCommand): Promise<IntegrationEntity[]> {
+    const integrations: IntegrationEntity[] = [];
+
+    if (
+      !process.env.NOVU_EMAIL_INTEGRATION_API_KEY ||
```

Comment:
we should not create any integrations if the credentials for them does not exists

Reply:
This also protects self hosted versions from getting these